editor/scene_tree_dock.cpp

```diff
@@ -1386,6 +1386,11 @@ void SceneTreeDock::_notification(int p_what) {
 
 			filter->set_right_icon(get_theme_icon(SNAME("Search"), SNAME("EditorIcons")));
 
+			PopupMenu *p_menu = filter->get_menu();
```

Comment:
```suggestion
			PopupMenu *filter_menu = filter->get_menu();
```
`p_` prefix should be used only for **p**arameters.